ExampleMod/Common/GlobalItems/ExampleExperienceItem.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Terraria;
+using Terraria.ModLoader;
+using Terraria.ID;
+using Microsoft.Xna.Framework;
+using Terraria.ModLoader.IO;
+
+namespace ExampleMod.Common.GlobalItems
+{
+	public class ExampleExperienceItem : GlobalItem {
+		public int experience;
+		public int level;
+		public static int experiencePerLevel = 100;
+
+		public override bool InstancePerEntity => true;
+
+		public override bool AppliesToEntity(Item entity, bool lateInstantiation) {
+			//Apply this GlobalItem to all swords
+			return entity.DamageType == DamageClass.Melee && entity.noMelee == false && entity.useStyle == ItemUseStyleID.Swing;
+		}
+
+		public override GlobalItem Clone(Item item, Item itemClone) {
```

Comment:
overriding clone is not required anymore for value typed fields, they will be cloned correctly automatically.